<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogStEnrolment</class>
 <widget class="QDialog" name="dialogStEnrolment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1047</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Student Enrolment</string>
  </property>
  <widget class="QGroupBox" name="gbEditDataStEnrolment">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>1011</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title">
    <string>Edit data</string>
   </property>
   <widget class="QLineEdit" name="txtEnrolStId">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>331</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Course code:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Student id:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Professor id:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>40</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Semester:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>90</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Term:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>140</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Enrolment date:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbEnrolCourseCode">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtEnrolProfessorId">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>331</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtEnrolSemester">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>40</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxEnrolTerm">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>90</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEditEnrolmentDate">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>140</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonEnrol">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>200</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>180</y>
      <width>171</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Radio buttons</string>
    </property>
    <widget class="QRadioButton" name="rbEnrolAdd">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Add enrolment</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbEnrolUpdate">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Update enrolment</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbEnrolDelete">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete enrolment</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbContentStEnrolment">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>1031</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>Content</string>
   </property>
   <widget class="QTableView" name="tvStEnrolment">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>1011</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
